src/components/specimen/components/contentBlocks/specimenBlock/Publisher.tsx

```diff
@@ -0,0 +1,108 @@
+                                <div className="w-0 b-accent" />
+                            </Col>
+                        </Row>
+                    </Col>
+                    <Col className="h-100 d-flex flex-column ps-0">
+                        {/* Block icon and title */}
+                        <Card.Title className="c-accent">
+                            <span> Publisher </span>
+                        </Card.Title>
+
+                        {/* Properties */}
+                        <Row className="mt-2 flex-grow-1 fs-4">
+                            <Col className="d-flex flex-column justify-content-between">
+                                {/* Organisation information */}
+                                <Row>
+                                    <Col>
+                                        {/* Organisation Name */}
+                                        <Row className="c-pointer"
+                                            onClick={() => ToggleSidePanel('dwc:institutionName')}
+                                        >
+                                            <Col className="col-md-auto pe-0 fw-lightBold">
+                                                Name:
+                                            </Col>
+                                            <Col>
+                                                <p> {specimen.digitalSpecimen['dwc:institutionName'] ?? 'Undefined'} </p>
+                                            </Col>
+                                        </Row>
+                                        {/*  */}
+                                        <Row className="c-pointer mt-2"
+                                            onClick={() => ToggleSidePanel('dwc:datasetName')}
+                                        >
+                                            <Col className="col-md-auto pe-0 fw-lightBold d-">
+                                                Collection:
+                                            </Col>
+                                            <Col className="d-flex align-items-center">
+                                                <p> {specimen.digitalSpecimen['dwc:datasetName'] ?? 'Undefined'} </p>
+                                            </Col>
+                                        </Row>
+                                        {/* Rights Holder */}
+                                        <Row className="c-pointer mt-2"
+                                            onClick={() => ToggleSidePanel('dcterms:rightsHolder')}
+                                        >
+                                            <Col className="col-md-auto pe-0 fw-lightBold">
+                                                Rights holder:
+                                            </Col>
+                                            <Col className="textOverfloww">
+                                                {specimen.digitalSpecimen['dcterms:rightsHolder'] ?? 'Undefined'}
```

Comment:
I think it should be left empty instead of "Undefined"

Reply:
Removed all the 'Undefined' strings